### PR TITLE
doku: Tab switchting podman docker

### PR DESCRIPTION
**Pull request by mand35**
### What changes are included in the pull request?

All docker src snippeds are now implemented with enabling the reader to select docker or podman commands. The commands are adapted to use `--pod` instead of `--network` where necessary.

### Why are the changes needed?
Now the user can easily copy and paste all commands ever if podman is used.

### CODE REVIEW

**Comment by mand35** [comment]
@Geheiner, thanks for the hint and the preparation. I just filled it in the remaining parts. Luckily the selection of `podman` or `docker` is remembered even when switching pages 👍 😃

File: docs/getting-started/part-1/select-columns.md [hunk 1]
```diff
@@ -67,7 +70,7 @@ Notice that we call our input DataObject stg-airports with a hyphen "-", but in
 This is due to the SQL standard not allowing "-" in unquoted identifiers (e.g. table names). 
 Under the hood, Apache Spark SQL is used to execute the query, which implements SQL standard.
 SDL works around this by replacing special chars in DataObject names used in SQL statements for you. 
-In this case, it automatically replaced "-" with "_"
+In this case, it automatically replaced "-" with `_`
```
Comment (Geheiner) [inline docs/getting-started/part-1/select-columns.md]:
```suggestion
In this case, it automatically replaced `-` with `_`
```

File: docs/getting-started/part-1/select-columns.md [hunk 2]
```diff
@@ -83,15 +86,37 @@ try out our new actions.
...
 If you encounter an error that looks like this:
 
-    Exception in thread "main" io.smartdatalake.util.dag.TaskFailedException: Task select-airport-cols failed. Root cause is 'IllegalArgumentException: requirement failed: (DataObject~stg-airports) DataObject schema is undefined. A sche
-    ma must be defined if there are no existing files.'
-    Caused by: java.lang.IllegalArgumentException: requirement failed: (DataObject~stg-airports) DataObject schema is undefined. A schema must be defined if there are no existing files.
+    Exception in thread "main" io.smartdatalake.util.dag.TaskFailedException: Task select-airport-cols failed. 
+			Root cause is 'IllegalArgumentException: requirement failed: (DataObject~stg-airports) DataObject schema 
+			is undefined. A schema must be defined if there are no existing files.'
+    Caused by: java.lang.IllegalArgumentException: requirement failed: (DataObject~stg-airports) DataObject 
+			schema is undefined. A schema must be defined if there are no existing files.
```
Comment (Geheiner) [inline docs/getting-started/part-1/select-columns.md]: why the change in formatting?

File: docs/getting-started/part-2/delta-lake-format.md [hunk 3]
```diff
@@ -145,11 +192,31 @@ This instructs Spark to use the external metastore you started with docker-compo
 Your Smart Data Lake container doesn't have access to the other containers just yet. 
 So when you run your data pipeline again, you need to add a parameter `--network getting-started_default` to join the virtual network where the metastore is located:
```
Comment (Geheiner) [inline docs/getting-started/part-2/delta-lake-format.md]:
```suggestion
So when you run your data pipeline again, you need to add a parameter to join the virtual network where the metastore is located:
```

File: docs/getting-started/part-2/delta-lake-format.md [hunk 4]
```diff
@@ -145,11 +192,31 @@ This instructs Spark to use the external metastore you started with docker-compo
...
-    podman run --hostname localhost --rm -v ${PWD}/data:/mnt/data -v ${PWD}/target:/mnt/lib -v ${PWD}/config:/mnt/config --pod getting-started sdl-spark:latest -c /mnt/config --feed-sel '.*'
+```jsx
+podman run --hostname localhost --rm -v ${PWD}/data:/mnt/data -v ${PWD}/target:/mnt/lib -v ${PWD}/config:/mnt/config --pod getting-started sdl-spark:latest -c /mnt/config --feed-sel '.*'
+```
+
+</TabItem>
+</Tabs>
+
+:::info Podman pod specification
+When using podman you need to join the pod where the metastore is located with `--pod getting-started`:
```
Comment (Geheiner) [inline docs/getting-started/part-2/delta-lake-format.md]: I suggest to remove this info section now

File: docs/getting-started/setup.md [hunk 5]
```diff
@@ -3,6 +3,9 @@ id: setup
 title: Technical Setup
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ## Requirements
 
 To run this tutorial you just need two things:
```
Comment (Geheiner) [inline docs/getting-started/setup.md:11]: I suggest to replace 
If you use Windows, please read our note on [Docker for Windows](troubleshooting/docker-on-windows.md),
with

If you use Windows, you might want to use [podman as an alternative to docker](troubleshooting/docker-on-windows.md).

**Review by Geheiner — CHANGES_REQUESTED** [review]
Thanks @mand35 ! I have some minor comments.

File: docs/getting-started/part-1/select-columns.md (hunk 1 above)
Comment (mand35) [reply docs/getting-started/part-1/select-columns.md]: Ahh indeed, more consistent. Thanks.

File: docs/getting-started/part-1/select-columns.md (hunk 2 above)
Comment (mand35) [reply docs/getting-started/part-1/select-columns.md]: I hoped to improve the readability.

File: docs/getting-started/part-2/delta-lake-format.md (hunk 3 above)
Comment (mand35) [reply docs/getting-started/part-2/delta-lake-format.md]: I would still mention which option we add. Changed the comment and added another commit. I changed it to:
```
So when you run your data pipeline again, you need to add a parameter `--network`/`--pod` to join the virtual network where the metastore is located:
```

File: docs/getting-started/part-2/delta-lake-format.md (hunk 4 above)
Comment (mand35) [reply docs/getting-started/part-2/delta-lake-format.md]: right, with the change in description above, this comment can be removed. I added another commit

File: docs/getting-started/setup.md (hunk 5 above)
Comment (mand35) [reply docs/getting-started/setup.md:11]: great suggestion. Thanks. Changed.

**Comment by mand35** [comment]
Thanks @Geheiner  for your great suggestions. I followed them, beside of the line wrapping one. We may want to discuss further.
